cpp/include/cudf/io/avro.hpp

```diff
@@ -73,21 +70,29 @@ class avro_reader_options {
 
   /**
    * @brief Returns source info.
+   *
+   * @return Source info
```

Comment:
`@return` basically repeats what's been explained in `@brief`. This kind of warning seems an "overkill" for simple getters like here.

Reply:
There is no way to differentiate between simple getters and legitimate `@return` missing.
at the moment, doxygen has no way to differentiate or simplify this. 

Aliases is one solution to reduce verbose typing. but not recommended so far in reviews https://github.com/rapidsai/cudf/pull/10861#pullrequestreview-975990743